libs/core/src/lib/panel/panel-header/panel-header.component.ts

```diff
@@ -1,26 +1,57 @@
-import { ChangeDetectionStrategy, Component, HostBinding, ViewEncapsulation } from '@angular/core';
+import {
+    AfterViewInit,
+    ChangeDetectionStrategy,
+    Component,
+    ContentChild,
+    EventEmitter,
+    HostBinding,
+    Output,
+    ViewEncapsulation
+} from '@angular/core';
+import { PanelExpandComponent } from '../panel-expand/panel-expand.component';
 
 /**
- * Header of the panel. Contains a head and actions.
+ * Applies the panel header style to a div element
+ * and serves as a container for the Panel Expand component, 
+ * the Panel Title and an optional Toolbar component.
  *
  * ```html
- * <fd-panel>
- *     <fd-panel-header>
- *         <fd-panel-head>
- *             <fd-panel-title>Title!</fd-panel-title>
- *         </fd-panel-head>
- *     </fd-panel-header>
- * </fd-panel>
+ * <div fd-panel-header>
+ *   <div fd-panel-expand></div>
+ *   <h5 fd-panel-title>Panel Header</h5>
+ *   <div>Other content</div>
+ * </div>
  * ```
  */
 @Component({
-    selector: 'fd-panel-header',
+    // tslint:disable-next-line:component-selector
+    selector: '[fd-panel-header]',
     templateUrl: './panel-header.component.html',
     encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class PanelHeaderComponent {
+export class PanelHeaderComponent implements AfterViewInit {
     /** @hidden */
     @HostBinding('class.fd-panel__header')
-    fdPanelHeaderClass = true;
+    readonly fdPanelHeaderClass: boolean = true;
+
+    /** 
+     * An event emitted to the parent PaneComponent 
+     * with a boolean value captured from the child PanelExpandComponent 
+     * that controls the expanded state of the Panel Content.
+     */
+    @Output()
+    expandedValue: EventEmitter<boolean> = new EventEmitter<boolean>();
```

Comment:
Could you change this event's naming to `expandedChange`

Reply:
Also let's have `expanded` flag instead of `isExpanded`. Compiled with `expandedChange` user will be able to use `[(expanded)]` binding, to change it programatically and react on changes.

Reply:
I will. That's a good idea. Thank you!